docs/scripts.js

```diff
@@ -31,14 +31,25 @@ var isLocal = window.location.hostname === 'localhost' || window.location.hostna
 // true if testing in an IP page URL
 var isIpAddress = /^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$/.test(window.location.hostname);
 
+var isNetlify = window.location.hostname.includes('cld-video-player.netlify.app');
+
 var cdnPrefix = function (source, ver) {
-  var external = source.startsWith('http');
+  var external = source && source.startsWith('http');
+  var previewBuild = ver && ver.startsWith('http');
+
+  if (previewBuild) {
+    return ver;
+  }
```

Comment:
not sure if infra will support this in this project, but basically you can remove the previewBuild function and just do 
```suggestion
  if (ver?.startsWith?.('http')) {
    return ver;
  }
```

Also i would rename ver to version